.gitignore

```diff
@@ -888,3 +888,6 @@ package
 
 ### Ignore environment file
 .env
+
+### VCPKG install
+vcpkg_installed/
```

Comment:
The `vcpkg_installed` should be going in the `build` folder, are you seeing otherwise? This shouldn't be needed